workflow-job

```diff
@@ -95,7 +95,7 @@ public class CommitStatus implements Serializable {
 }
 
 public class Images implements Serializable {
-    def cassandra = 'cassandra'
+    def cassandra = 'cassandra:2.1.9'
```

Comment:
When updating Cassandra unit, I get these strange errors : 

```
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] no libsigar-amd64-linux.so in java.library.path
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] org.hyperic.sigar.SigarException: no libsigar-amd64-linux.so in java.library.path
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.hyperic.sigar.Sigar.loadLibrary(Sigar.java:172)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.hyperic.sigar.Sigar.<clinit>(Sigar.java:100)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.apache.cassandra.utils.SigarLibrary.<init>(SigarLibrary.java:45)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.apache.cassandra.service.StartupChecks$5.execute(StartupChecks.java:178)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.apache.cassandra.service.StartupChecks.verify(StartupChecks.java:103)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.apache.cassandra.service.CassandraDaemon.setup(CassandraDaemon.java:162)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:511)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at org.cassandraunit.utils.EmbeddedCassandraServerHelper$1.run(EmbeddedCassandraServerHelper.java:123)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] 	at java.lang.Thread.run(Thread.java:745)
```

As it is not directly linked to MAILBOX-266 ticket, I reset my branch and will not investigate on this much...